### PR TITLE
Transitions

### PR DESCRIPTION
**Pull request by gyosida**

### CODE REVIEW

**Comment by gyosida** [comment]
@cpinan @PedroCarrillo take a look

File: app/src/main/java/com/belatrixsf/allstars/adapters/MainNavigationViewPagerAdapter.java [hunk 1]
```diff
@@ -55,6 +53,8 @@ public Fragment getItem(int position) {
         }
     }
 
+
+
```
Comment (PedroCarrillo) [inline app/src/main/java/com/belatrixsf/allstars/adapters/MainNavigationViewPagerAdapter.java]: ????

File: app/src/main/java/com/belatrixsf/allstars/ui/ranking/RankingFragment.java [hunk 2]
```diff
@@ -93,8 +96,9 @@ public void onViewCreated(View view, Bundle savedInstanceState) {
     private void initViews() {
         rankingListAdapter = new RankingListAdapter(this);
         rankingRecyclerView.setAdapter(rankingListAdapter);
-        rankingRecyclerView.setNestedScrollingEnabled(false);
+        rankingRecyclerView.setNestedScrollingEnabled(false);
```
Comment (PedroCarrillo) [inline app/src/main/java/com/belatrixsf/allstars/ui/ranking/RankingFragment.java]: ?

Comment (neosergio) [inline app/src/main/java/com/belatrixsf/allstars/ui/ranking/RankingFragment.java]: ?

Comment (gyosida) [inline app/src/main/java/com/belatrixsf/allstars/ui/ranking/RankingFragment.java]: ?

File: app/src/main/java/com/belatrixsf/allstars/adapters/MainNavigationViewPagerAdapter.java (hunk 1 above)
Comment (gyosida) [inline app/src/main/java/com/belatrixsf/allstars/adapters/MainNavigationViewPagerAdapter.java]: !!!!

File: app/src/main/java/com/belatrixsf/allstars/ui/ranking/RankingFragment.java (hunk 2 above)
Comment (PedroCarrillo) [inline app/src/main/java/com/belatrixsf/allstars/ui/ranking/RankingFragment.java]: After extensive reviewing and discussion, I've came along an extra character placed at the end of the semicolons in line 99. Although this has not a direct impact on code functionality, it affects the base coding guidelines we are following.

Comment (cpinan) [inline app/src/main/java/com/belatrixsf/allstars/ui/ranking/RankingFragment.java]: ?

File: app/src/main/java/com/belatrixsf/allstars/adapters/MainNavigationViewPagerAdapter.java (hunk 1 above)
Comment (cpinan) [inline app/src/main/java/com/belatrixsf/allstars/adapters/MainNavigationViewPagerAdapter.java]: D=

File: app/src/main/java/com/belatrixsf/allstars/ui/account/AccountFragment.java [hunk 3]
```diff
@@ -208,7 +211,35 @@ public void showSkypeId(String skypeID) {
...
+                        return false;
+                    }
+
+                    @Override
+                    public boolean onResourceReady(GlideDrawable resource, String model, Target<GlideDrawable> target, boolean isFromMemoryCache, boolean isFirstResource) {
+                        startPostponedEnterTransition();
+                        return false;
+                    }
+                })
+                .into(pictureImageView);
```
Comment (PedroCarrillo) [inline app/src/main/java/com/belatrixsf/allstars/ui/account/AccountFragment.java:231]: @cpinan I think you have to take this in consideration when refactoring image loader

Comment (cpinan) [inline app/src/main/java/com/belatrixsf/allstars/ui/account/AccountFragment.java:231]: I did not see that piece of code. I am going to add the implementation in the loader.

File: app/src/main/java/com/belatrixsf/allstars/adapters/MainNavigationViewPagerAdapter.java (hunk 1 above)
Comment (icerrate) [inline app/src/main/java/com/belatrixsf/allstars/adapters/MainNavigationViewPagerAdapter.java]: :v

**Comment by PedroCarrillo** [comment]
close #43

**Comment by PedroCarrillo** [comment]
close #43

**Pull request merged**